server/db/user.js

```diff
@@ -0,0 +1,60 @@
+const Sequelize = require('sequelize');
+const db = require('./connection');
+
+const User = db.define('user', {
+  id: {
+    type: db.Sequelize.UUID,
+    defaultValue: db.Sequelize.UUIDV4,
+    unique: true,
+    primaryKey: true,
+  },
+  firstName: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notNull: true,
```

Comment:
`allowNull` and `validate.notNull` are the same thing. Repeat for all following instances.

Reply:
I think notEmpty is what you might have been going for. Validate against an empty string.

Reply:
Thanks for the feedback. I did mean notEmpty, I'll fix that